.github/workflows/main.yml

```diff
@@ -77,15 +77,15 @@ jobs:
             publish: true
 
           - name: windows-python3
-            os: windows-2019
+            os: windows-2016
```

Comment:
Just so I understand, is this because the dependencies are build with MSVC 2017 rather than 2019?

Reply:
Yes, the dependencies are built on MSVC 2017 and I was getting weird test failures in USD (bc860c46a80301e07107b75928b179a083badabc) when mixing compiler versions.